bpkg.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "bashunit",
+  "description": "A simple testing library for bash scripts.",
+  "scripts": [ "bashunit" ],
+  "install": "make install",
```

Comment:
Be careful with this, make does not work on windows

Reply:
Actually, I was wondering about this "install" key. Is it really [necessary](https://bpkg.sh/guidelines/#install)? What should we put there?  🤔

Reply:
Actually, I was wondering about this "install" key. Is it really [necessary](https://bpkg.sh/guidelines/#install)?
What should we put there, @samlikins?  🤔

Reply:
I'm guessing that it's the script that will install our software, but I'm not sure

Reply:
The `scripts` property is an array of paths (to files in the project) which you want installed in the `deps` directory, when your project is included as a dependency.  When installing globally, the `install` property is a command string for performing a system install.  The example uses a `Makefile`, but it could be a shell script in your project.  Take a look at some of the other projects.  The `bash-util-lib` project uses an `install` command.  The `dots` project uses a shell script.  The `JSON` project uses a simple `cp` command.  Using the `PREFIX` environment variable will provide a system `bin` parent directory, if performing an install as `root`, a user directory otherwise.

Reply:
The `global` property should be removed, if you want it to be installed in projects, instead of system wide.  Try the following for the install property: `install -b bashunit.sh ${PREFIX:-/usr/local}/bin/bashunit`, this will be run when someone wants to install as a system executable.

Reply:
Great. I will try that for now. Thank you!